rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md

```diff
@@ -141,6 +151,16 @@ Nervos DAO script is for **type script**:
 
 Simple UDT script is for **type script**:
 
+- Lina
+
+| parameter   | value                                                                |
+| ----------- | -------------------------------------------------------------------- |
+| `code_hash` | `0x5e7a36a77e68eecc013dfa2fe6a23f3b6c344b04005808694ae6dd45eea4cfd5` |
+| `hash_type` | `type`                                                               |
+| `tx_hash`   | `0xc7813f6a415144643970c2e88e0bb6ca6a8edc5dd7c1022746f628284a9936d5` |
```

Comment:
The cell 0xc7813f6a415144643970c2e88e0bb6ca6a8edc5dd7c1022746f628284a9936d5.outputs[0x0] matches the information in the [sUDT RFC](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0025-simple-udt/0025-simple-udt.md#notes), which type is

```
{
    "args": "0xcf37df53ec0eb890937879ad5587521f737dc3d95eb1c22194b958ecbcf39a8b",
    "codeHash": "0x00000000000000000000000000000000000000000000000000545950455f4944",
    "hashType": "type"
}
```

Its type hash is expected to be `0x5e7a36a77e68eecc013dfa2fe6a23f3b6c344b04005808694ae6dd45eea4cfd5` and can be verified by

```
echo '{ "args": "0xcf37df53ec0eb890937879ad5587521f737dc3d95eb1c22194b958ecbcf39a8b", "code_hash": "0x00000000000000000000000000000000000000000000000000545950455f4944", "hash_type": "type" }' > 0x5e7a36a77e68eecc013dfa2fe6a23f3b6c344b04005808694ae6dd45eea4cfd5.json
ckb-cli molecule encode --json-path 0x5e7a36a77e68eecc013dfa2fe6a23f3b6c344b04005808694ae6dd45eea4cfd5.json --type Script --output-type hash
```